### ISSUE TITLE
Question: animations

### ISSUE THREAD

**Issue by ganySA** [Breeze/breeze.js#169]
Hi

Problem: trying to animate lists of data.

I am wondering if there is an approach to animations and breeze in angular2 specifically. The only way i can think of myself is by pushing individual breeze entities into another array and doing animations that way in an ngfor loop?

Thanks

**Comment by marcelgood** [comment]
No, there isn't anything specifically to breeze as far as animations are concerned. That's a concern for the UI and not the data layer.
